src/Wsdl/WebServiceName.php

```diff
@@ -63,9 +63,9 @@ abstract class WebServiceName
 
 
   /**
-   * @return string[]
+   * @return string[]s
    */
-  private static function getConstants(): array
+  public static function getConstants(): array
```

Comment:
Zvážil bych jestli nepoužít popisnější název, když je to teď public API. Alespoň něco jako `getAll()`.